tools/azsdk-cli/Azure.Sdk.Tools.Cli/Configuration/SdkRepoConfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "_description": "Configuration file for SDK repository settings. Defines the relative path to the 'swagger_to_sdk_config.json' file in SDK language repos. The 'swagger_to_sdk_config.json' file contains language-specific settings for init, generate, and package process.",
+  "_usage": "This file is used by SdkBuildTool and SdkPostGenerateTool to determine how to build and post-generate SDK packages for different languages.",
+  "azure-sdk-for-go": {
+    "configFilePath": "eng/swagger_to_sdk_config.json"
```

Comment:
not related to your PR, but is there a reason why every repo has this file in different locations?

Reply:
Could we also just discover this file? We could hardcode these three locations and just look for each one, then we don't need this json config.

Reply:
We were pushing each language repo to have same location of this configuration file. However, we used to have two pipeline approaches which relied on different version of these config files. And moving the file involves the existed code migration.

I agree we could take this chance to unify them. I'll add the config file to `\eng` for each SDK repos so that this code base can be simplified. Will create task for SDK validation pipeline to use these configs from the new location.